projects/fsastorefrontlib/src/cms-components/my-account/claim/claim-details/claim-details.component.html

```diff
@@ -0,0 +1,145 @@
+            {{ claim.claimNumber }}
+          </div>
+        </li>
+        <li class="accordion-list-item row">
+          <div class="col-6 col-sm-5 item-label">
+            {{ 'policy.policyNo' | cxTranslate }}
+          </div>
+          <div class="col-6 col-sm-5 value notice">
+            {{ claim.insurancePolicy?.policyNumber }}
+          </div>
+        </li>
+        <li class="accordion-list-item row">
+          <div class="col-6 col-sm-5 item-label">
+            {{ 'claim.dateOfLoss' | cxTranslate }}
+          </div>
+          <div class="col-6 col-sm-5 value notice">
+            <ng-container *ngIf="claim.dateOfLoss != undefined; else na">
+              {{ claim.dateOfLoss | parseDate | cxDate: 'dd MMM yyyy' }}
+            </ng-container>
+          </div>
+        </li>
+        <li class="accordion-list-item row">
+          <div class="col-6 col-sm-5 item-label">
+            {{ 'claim.claimHandler' | cxTranslate }}
+          </div>
+          <div class="col-6 col-sm-5 value notice">
+            <ng-container
+              *ngIf="claim.claimHandler?.displayName != undefined; else na"
+            >
+              {{ claim.claimHandler.displayName }}</ng-container
+            >
+          </div>
+        </li>
+        <li class="accordion-list-item row">
+          <div class="col-6 col-sm-5 item-label">
+            {{ 'fscommon.emailAddress' | cxTranslate }}
+          </div>
+          <div class="col-6 col-sm-5 value notice">
+            <ng-container
+              *ngIf="claim.claimHandler?.email != undefined; else na"
+              >{{ claim.claimHandler.email }}</ng-container
+            >
+          </div>
+        </li>
+        <li class="accordion-list-item row">
+          <div class="col-6 col-sm-5 item-label">
+            {{ 'fscommon.status' | cxTranslate }}
+          </div>
+          <div class="col-6 col-sm-5 value notice">
+            <ng-container *ngIf="claim.claimStatus != undefined; else na">
+              {{ claim.claimStatus }}
+            </ng-container>
+          </div>
+        </li>
+      </ul>
+    </cx-fs-accordion-item>
+    <cx-fs-accordion-item
+      *ngIf="claim.documents && claim.documents.length > 0"
+      [title]="'fscommon.documents' | cxTranslate"
+    >
+      <ul class="accordion-list container-fluid documents">
+        <li class="accordion-list-item row">
+          <div class="col-3">
+            {{ 'policy.documentName' | cxTranslate }}
+          </div>
+          <div class="col-3 value">
+            {{ 'claim.created' | cxTranslate }}
+          </div>
+          <div class="col-3 value">
+            {{ 'claim.documentType' | cxTranslate }}
+          </div>
+          <div class="col-3 value">
+            {{ 'claim.uploadedByMe' | cxTranslate }}
+          </div>
+        </li>
+        <ng-container *ngFor="let document of claim.documents">
+          <li class="accordion-list-item row">
+            <div class="col-3">
+              <a
```

Comment:
Maybe replace `a` for `span` tag, and just add a class `notice` if I remember correctly to emphasize that it's clickable. I don't see any benefit from anchor tag in this context.  You wouldn't need `prevent.default()` in that case either.